tools/travis.sh

```diff
@@ -1,7 +1,10 @@
 set -x
 ulimit -n 4096
+# do this for our daily build test run
+if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
```

Comment:
Maybe we can leave some other hook to allow full builds on CI with [REST api](https://docs.travis-ci.com/api#making-requests).